spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/GatewayApiController.java

```diff
@@ -51,4 +65,19 @@ public ApiKey findApiKeyEntityByApiKeyValue(@RequestParam("keyValue") String api
     public ServiceApi findServiceByApiKeyValue(@RequestParam("apiKeyValue") String apiKeyValue) {
         return serviceApiRepository.findByApiKeyValue(apiKeyValue);
     }
+
+    @GetMapping( "/api/gateways/{gatewayId}/receiveApiKeyList")
```

Comment:
Here the name in the path is "receiveApiKeyList", shouldn't it be send the API Key list (as on the admin side, the admin is sending the API Key List to the Gateway)

Reply:
Solved.

Reply:
I checked the new version of the branch and the name is still "receiveApiKeyList", maybe you forget to push the last version of the branch from your local

Reply:
Suggestion to follow REST-naming best practices:
```suggestion
    @GetMapping( "/api/gateways/{gatewayId}/apikeys")
```

Reply:
Indeed I would prefer that name, just to avoid misleading when reading the code

Reply:
You are right, I forgot to perform the push. Now the latest version should be available. The URL suggestion was incorporated.